src/site/stages/build/drupal/graphql/paragraph-fragments/qa.paragraph.graphql.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * A Drupal paragraph for questions and answers. Answers come in the format of WYSIWYG, collapsible panel, and process.
+ *
+ */
+module.exports = `
+	fragment qa on ParagraphQA {
```

Comment:
Extra indentation here

Reply:
Sorry about that (tab instead of spaces). Fixed now.